source/features/render-conventional-commit-types.tsx

```diff
@@ -0,0 +1,82 @@
+import './render-conventional-commit-types.css';
+import React from 'react';
+import {$} from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+import {commitTitleInLists} from '../github-helpers/selectors.js';
+import {getConventionalCommitAndScopeMatch, removeCommitAndScope} from '../helpers/render-conventional-commit-types.js';
+
+const types = new Map([
+	['feat', 'Feature'],
+	['fix', 'Fix'],
+	['chore', 'Chore'],
+	['docs', 'Docs'],
+	['build', 'Build'],
+	['refactor', 'Refactor'],
+	['test', 'Test'],
+	['ci', 'CI'],
+	['perf', 'Performance'],
+]);
+
+function createLabelElement(type: string, scope?: string): JSX.Element {
+	const label = types.get(type)!;
+
+	return (
+		<span className={`IssueLabel hx_IssueLabel rgh-commit-type-label rgh-commit-type-label-${type}-colors`}>
+			{scope ? `${label}: ${scope}` : label}
+		</span>
+	);
+}
+
+function renderLabelInCommitTitle(commitTitleElement: HTMLElement): void {
+	const match = getConventionalCommitAndScopeMatch(commitTitleElement.textContent);
+
+	const {type, scope} = match?.groups ?? {};
+	if (!type || !types.has(type)) {
+		return;
+	}
+
+	removeCommitAndScope(commitTitleElement, match!);
+	commitTitleElement.prepend(createLabelElement(type, scope));
+}
+
+function renderLabelInRepoCommitTitle(commitTitleElement: HTMLElement): void {
+	const repoCommitTitleElement = $('h4 > span', commitTitleElement)!;
+	renderLabelInCommitTitle(repoCommitTitleElement);
+}
+
+function renderLabelInPRCommitTitle(commitTitleElement: HTMLElement): void {
```

Comment:
These two functions should be dropped. The observed selector already points to the final element. 

Pass `renderLabelInCommitTitle` directly to the observer